tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -42,7 +42,7 @@ using namespace ROOT::Experimental;
 using namespace ROOT::Experimental::Internal;
```

Comment:
```suggestion
using namespace ROOT;
using namespace ROOT::Experimental::Internal;
```
(and keeping the rest of the code as is).
seems to match 'better' the spirit of this file coding style.

Reply:
I think the idea of `using namespace ROOT::Experimental::Internal` was avoiding long boilerplate strings throughout the file. However, I find the `ROOT::` prefix short and concise and I'd have a preference of writing `ROOT::RNTuple` rather than `RNTuple` (similar to `std::string` over `string`).